### PR TITLE
[F#] Fixes interactive window. Fixes #52721

### PR DESCRIPTION
**Pull request by nosami**
Fixes https://bugzilla.xamarin.com/show_bug.cgi?id=52721

### CODE REVIEW

**Comment by nosami** [comment]
Waiting for a review. It's [passed QA](https://trello.com/c/CYToXPaj)

**Comment by nosami** [comment]
Thanks @Therzok !

File: main/external/fsharpbinding/MonoDevelop.FSharpBinding/FSharpInteractivePad.fs [hunk 1]
```diff
@@ -189,7 +189,8 @@ type FSharpInteractivePad() =
 
     let setupSession() =
         try
-            let ses = InteractiveSession()
+            let pathToExe = "\"" + Path.Combine(Reflection.Assembly.GetExecutingAssembly().Location |> Path.GetDirectoryName, "MonoDevelop.FSharpInteractive.Service.exe") + "\""
```
Comment (Therzok) [inline main/external/fsharpbinding/MonoDevelop.FSharpBinding/FSharpInteractivePad.fs]: Just for future notice: `ProcessArgumentBuilder.Quote` exists and should have better perf.

Comment (nosami) [inline main/external/fsharpbinding/MonoDevelop.FSharpBinding/FSharpInteractivePad.fs]: Cheers. [Fixed in vNext ](https://github.com/mono/monodevelop/commit/98dd1776afcf09abadd7c96f7e6bde245a8b0a4c)

**Pull request merged**